src/components/file-upload/index.md.njk

```diff
@@ -19,7 +19,7 @@ You should only ask users to upload something if it’s critical to the delivery
 
 ## How it works
 
-There are 2 ways to use the File upload component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com), you can use the File upload macro.
+There are 2 ways to use the File upload component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com), you can use the File Nunjucks macro.
```

Comment:
Replace `you can use the File Nunjucks macro` with `you can use the Nunjucks macro`